src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AzureFunction/InvokeFunctionAsyncWrapper.cs

```diff
@@ -96,6 +105,32 @@ void InvokeFunctionAsyncResponse(Task responseTask)
                         transaction.NoticeError(responseTask.Exception);
                         return;
                     }
+
+                    // only pull response status code here if it's a web trigger and the Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore assembly is not loaded.
+                    if (functionDetails.IsWebTrigger && functionDetails.HasAspNetCoreExtensionReference != null && !functionDetails.HasAspNetCoreExtensionReference.Value)
+                    {
+                        if (_getInvocationResultMethod == null)
+                        {
+                            // GetInvocationResult is a static extension method
+                            // there are multiple GetInvocationResult methods in this type; we want the one without any generic parameters
+                            Type type = functionContext.GetType().Assembly.GetType("Microsoft.Azure.Functions.Worker.FunctionContextBindingFeatureExtensions");
```

Comment:
Looks like we're referencing a few long class/type names. Maybe it would be cleaner to make them constants?